homeassistant/components/spotify/strings.json

```diff
@@ -1,14 +1,16 @@
 {
   "config": {
     "step": {
-      "pick_implementation": { "title": "[%key:common::config_flow::title::oauth2_pick_implementation%]" },
+      "pick_implementation": {
+        "title": "[%key:common::config_flow::title::oauth2_pick_implementation%]"
+      },
       "reauth_confirm": {
         "title": "Re-authenticate with Spotify",
         "description": "The Spotify integration needs to re-authenticate with Spotify for account: {account}"
       }
     },
     "abort": {
-      "already_setup": "You can only configure one Spotify account.",
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
```

Comment:
I have gone through the codebase, this key is not in use for this flow, lets just remove it instead.

Reply:
Yeah I have looked through the config flow as well and was confused when I did not find it. Will remove it asap.